features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,46 @@
+Given(/^I am on login page$/) do
+  visit LoginPage
+end
+
+When(/^I click at arrow button$/) do
+  on(LoginPage) do |login_page|
+    login_page.access_to_another_acc_element.click if login_page.access_to_another_acc_element.exists?
+  end
+end
+
+When(/^I click at Use another account button$/) do
+  on(LoginPage) do |login_page|
+    login_page.use_another_account_element.click if login_page.use_another_account_element.exists?
+  end
+end
+
+And(/^I enter valid email/) do
+  on(LoginPage) do |login_page|
+    login_page.login_field_element.click
+    login_page.login_field_element.send_keys "rubyautomationtraining@gmail.com"
+  end
+end
+
+And(/^I click Next button$/) do
+  on(LoginPage) do |login_page|
+    if login_page.next_button_element.exists?
+      login_page.next_button_element.element.wd.click
+    else
+      login_page.password_next_button_element.element.wd.click
+    end
+  end
+end
+
+And(/^I enter valid password$/) do
+  on(LoginPage) do |login_page|
+    login_page.password_field_element.send_keys "1qaz!QAZ1q"
+  end
+end
+
+Then(/^I logged in$/) do
+  sleep 5
```

Comment:
Sometimes when_present method doesn't work properly on few elements and i cannot locate element before page is loaded.